pkg/kudo/errors.go

```diff
@@ -0,0 +1,25 @@
+package kudo
+
+import (
+	"fmt"
+
+	kudov1beta1 "github.com/kudobuilder/kudo/pkg/apis/kudo/v1beta1"
+)
+
+// PlanStatusTimeout is the error returned when waiting for a plan status times out.
+type PlanStatusTimeout struct {
+	Plan           string
+	ExpectedStatus kudov1beta1.ExecutionStatus
+	ActualStatus   kudov1beta1.ExecutionStatus
```

Comment:
I would really like to have the PlanStatus.message included in here, that would help the most with debugging a failing test.

Reply:
Makes sense, added it.